.github/workflows/build-test.yml

```diff
@@ -230,20 +230,18 @@ jobs:
           benchmark-data-dir-path: ./benchmarks/perf/${{ matrix.component }}
           # Show alert with commit comment on detecting possible performance regression
           alert-threshold: '200%'  # If for nothing else, enabling the possibility of alerts with meaningful thresholds requires this job to be done per-component
-          fail-on-alert: true
```

Comment:
What does this line mean?  It seems like it would still be useful to have the CI job fail if it found a regression.

Reply:
I agree.